tests/security/TopicWithoutPermissions/TopicWithoutPermissions_export.h

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+// $Id$
+// Definition for Win32 Export directives.
+// This file is generated automatically by generate_export_file.pl TopicWithoutPermissions
+// ------------------------------
+#ifndef TOPICWITHOUTPERMISSIONS_EXPORT_H
+#define TOPICWITHOUTPERMISSIONS_EXPORT_H
+
+#include "ace/config-all.h"
+
+#if defined (ACE_AS_STATIC_LIBS) && !defined (TOPICWITHOUTPERMISSIONS_HAS_DLL)
+#  define TOPICWITHOUTPERMISSIONS_HAS_DLL 0
+#endif /* ACE_AS_STATIC_LIBS && TOPICWITHOUTPERMISSIONS_HAS_DLL */
+
+#if !defined (TOPICWITHOUTPERMISSIONS_HAS_DLL)
+#  define TOPICWITHOUTPERMISSIONS_HAS_DLL 1
+#endif /* ! TOPICWITHOUTPERMISSIONS_HAS_DLL */
+
+#if defined (TOPICWITHOUTPERMISSIONS_HAS_DLL) && (TOPICWITHOUTPERMISSIONS_HAS_DLL == 1)
+#  if defined (TOPICWITHOUTPERMISSIONS_BUILD_DLL)
+#    define TopicWithoutPermissions_Export ACE_Proper_Export_Flag
+#    define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARATION(T) ACE_EXPORT_SINGLETON_DECLARATION (T)
+#    define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARE(SINGLETON_TYPE, CLASS, LOCK) ACE_EXPORT_SINGLETON_DECLARE(SINGLETON_TYPE, CLASS, LOCK)
+#  else /* TOPICWITHOUTPERMISSIONS_BUILD_DLL */
+#    define TopicWithoutPermissions_Export ACE_Proper_Import_Flag
+#    define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARATION(T) ACE_IMPORT_SINGLETON_DECLARATION (T)
+#    define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARE(SINGLETON_TYPE, CLASS, LOCK) ACE_IMPORT_SINGLETON_DECLARE(SINGLETON_TYPE, CLASS, LOCK)
+#  endif /* TOPICWITHOUTPERMISSIONS_BUILD_DLL */
+#else /* TOPICWITHOUTPERMISSIONS_HAS_DLL == 1 */
+#  define TopicWithoutPermissions_Export
+#  define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARATION(T)
+#  define TOPICWITHOUTPERMISSIONS_SINGLETON_DECLARE(SINGLETON_TYPE, CLASS, LOCK)
+#endif /* TOPICWITHOUTPERMISSIONS_HAS_DLL == 1 */
+
+// Set TOPICWITHOUTPERMISSIONS_NTRACE = 0 to turn on library specific tracing even if
+// tracing is turned off for ACE.
+#if !defined (TOPICWITHOUTPERMISSIONS_NTRACE)
+#  if (ACE_NTRACE == 1)
+#    define TOPICWITHOUTPERMISSIONS_NTRACE 1
+#  else /* (ACE_NTRACE == 1) */
+#    define TOPICWITHOUTPERMISSIONS_NTRACE 0
+#  endif /* (ACE_NTRACE == 1) */
+#endif /* !TOPICWITHOUTPERMISSIONS_NTRACE */
+
+#if (TOPICWITHOUTPERMISSIONS_NTRACE == 1)
+#  define TOPICWITHOUTPERMISSIONS_TRACE(X)
+#else /* (TOPICWITHOUTPERMISSIONS_NTRACE == 1) */
+#  if !defined (ACE_HAS_TRACE)
+#    define ACE_HAS_TRACE
+#  endif /* ACE_HAS_TRACE */
+#  define TOPICWITHOUTPERMISSIONS_TRACE(X) ACE_TRACE_IMPL(X)
+#  include "ace/Trace.h"
+#endif /* (TOPICWITHOUTPERMISSIONS_NTRACE == 1) */
+
+#endif /* TOPICWITHOUTPERMISSIONS_EXPORT_H */
+
+// End of auto generated file.
```

Comment:
```suggestion
// End of auto generated file.

```

Reply:
The fuzz test is failing because of lacking a newline at the end of last line of file.